### PR TITLE
events/pubsub: google pub/sub event writer

### PR DESCRIPTION
**Pull request by buchanae**

### CODE REVIEW

**Comment by buchanae** [comment]
#453 adds a docker container for Google emulators, which is much easier/cleaner. When that's merged, I'll fix this up.

File: config/config.go [hunk 1]
```diff
@@ -166,6 +167,12 @@ type Kafka struct {
 	Topic   string
 }
 
+// PubSub configures access to Google Cloud Pub/Sub for task event reading/writing.
+type PubSub struct {
+	Topic   string
+	Project string
```
Comment (adamstruck) [inline config/config.go]: This should also have `CredentialsFile string` to be consistent with the other google components.

Comment (buchanae) [reply config/config.go]: Fixed in https://github.com/ohsu-comp-bio/funnel/pull/465/commits/8a7743b41a4f0e9bffb88fcb6be190bbb651efc5

Comment (adamstruck) [reply config/config.go]: It doesn't look like your `NewPubSubWriter` and `ReadPubSub` methods make use of the new field.

**Pull request merged**